pyproject.toml

```diff
@@ -15,22 +15,27 @@ force_single_line = true
 line_length = 80
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.11.2"
 astral = "^2.2"
 google-api-python-client = "^1.12.8"
 google-auth-httplib2 = "^0.0.4"
 google-auth-oauthlib = "^0.4.2"
 pydantic = "^1.7.3"
 click = "^8.0.1"
+rsa = "*"
```

Comment:
Why do we need `rsa`?

Reply:
I didn't know how to fix the error below without adding the package rsa. If there is another way, I'd like to know about it.

```
   from suncal.models.googlecal import GoogleCalEvent
../suncal/models/googlecal.py:7: in <module>
    from google.oauth2.credentials import Credentials
../../../Library/Caches/pypoetry/virtualenvs/suncal-1ZS7BLU5-py3.11/lib/python3.11/site-packages/google/oauth2/credentials.py:45: in <module>
    from google.oauth2 import reauth
../../../Library/Caches/pypoetry/virtualenvs/suncal-1ZS7BLU5-py3.11/lib/python3.11/site-packages/google/oauth2/reauth.py:40: in <module>
    from google.oauth2 import _client
../../../Library/Caches/pypoetry/virtualenvs/suncal-1ZS7BLU5-py3.11/lib/python3.11/site-packages/google/oauth2/_client.py:36: in <module>
    from google.auth import jwt
../../../Library/Caches/pypoetry/virtualenvs/suncal-1ZS7BLU5-py3.11/lib/python3.11/site-packages/google/auth/jwt.py:57: in <module>
    from google.auth import _service_account_info
../../../Library/Caches/pypoetry/virtualenvs/suncal-1ZS7BLU5-py3.11/lib/python3.11/site-packages/google/auth/_service_account_info.py:22: in <module>
    from google.auth import crypt
../../../Library/Caches/pypoetry/virtualenvs/suncal-1ZS7BLU5-py3.11/lib/python3.11/site-packages/google/auth/crypt/__init__.py:43: in <module>
    from google.auth.crypt import rsa
../../../Library/Caches/pypoetry/virtualenvs/suncal-1ZS7BLU5-py3.11/lib/python3.11/site-packages/google/auth/crypt/rsa.py:27: in <module>
    from google.auth.crypt import _python_rsa
../../../Library/Caches/pypoetry/virtualenvs/suncal-1ZS7BLU5-py3.11/lib/python3.11/site-packages/google/auth/crypt/_python_rsa.py:28: in <module>
    import rsa  # type: ignore
E   ModuleNotFoundError: No module named 'rsa'
```